docs/_docs/export.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Export Project to CLI
-order: 20
+order: 21
```

Comment:
in order to put the new doc in the right spot, all the others had to move. The other changes in each file are `prettier`